diff --git a/Makefile b/Makefile
index d3a6d641cf..16691bd66a 100644
--- a/Makefile
+++ b/Makefile
@@ -172,7 +172,10 @@ unix-style:
 plain-unix-style:
 	if [ "$(PREFIX)" = "" ] ; then $(MAKE) error-need-prefix ; fi
 	$(MAKE) $(VM)-base MORE_CONFIGURE_ARGS="$(MORE_CONFIGURE_ARGS) --prefix=$(PREFIX) --enable-macprefix" CONFIG_IN_PLACE_ARGS="" SELF_ROOT_CONFIG_FLAG="-Z" SKIP_DESTDIR_FIX="skip" AS_IS="$(AS_IS)" CONFIGURE_ARGS="$(CONFIGURE_ARGS)" CS_CONFIGURE_ARGS="$(CS_CONFIGURE_ARGS)" CS_HOST_WORKAREA_PREFIX="$(CS_HOST_WORKAREA_PREFIX)" EXTRA_REPOS_BASE="$(EXTRA_REPOS_BASE)" JOB_OPTIONS="$(JOB_OPTIONS)" PB_REPO="$(PB_REPO)" PLT_SETUP_OPTIONS="$(PLT_SETUP_OPTIONS)" RACKETBC_SUFFIX="$(RACKETBC_SUFFIX)" RACKETCS_SUFFIX="$(RACKETCS_SUFFIX)" RACKET_FOR_BOOTFILES="$(RACKET_FOR_BOOTFILES)" SETUP_MACHINE_FLAGS="$(SETUP_MACHINE_FLAGS)" SINGLE_BRANCH_FLAG="$(SINGLE_BRANCH_FLAG)"
-	$(MAKE) set-src-catalog DEFAULT_SRC_CATALOG="$(DEFAULT_SRC_CATALOG)" DESTDIR="$(DESTDIR)" PREFIX="$(PREFIX)" SRC_CATALOG="$(SRC_CATALOG)"
+	"$(DESTDIR)$(PREFIX)/bin/raco" pkg config -i --set catalogs "wedonotneednocatalogs"
+	sed -s "s/\"wedonotneednocatalogs\"//g" < "$(DESTDIR)$(PREFIX)/etc/racket/config.rktd" > "$(DESTDIR)$(PREFIX)/etc/racket/config.rktd1"
+	mv "$(DESTDIR)$(PREFIX)/etc/racket/config.rktd1" "$(DESTDIR)$(PREFIX)/etc/racket/config.rktd"
+	#	$(MAKE) set-src-catalog DEFAULT_SRC_CATALOG="$(DEFAULT_SRC_CATALOG)" DESTDIR="$(DESTDIR)" PREFIX="$(PREFIX)" SRC_CATALOG="$(SRC_CATALOG)"
 	$(MAKE) local-catalog DESTDIR="$(DESTDIR)" PREFIX="$(PREFIX)"
 	"$(DESTDIR)$(PREFIX)/bin/raco" pkg install $(JOB_OPTIONS) --catalog build/local/catalog --auto -i $(REQUIRED_PKGS) $(PKGS)
 	cd racket/src/build && $(MAKE) fix-paths
@@ -310,11 +313,11 @@ maybe-fetch-pb-as-is:
 	echo done
 fetch-pb-from:
 	mkdir -p racket/src/ChezScheme/boot
-	if [ ! -d racket/src/ChezScheme/boot/pb ] ; 	  then git clone -q $(SINGLE_BRANCH_FLAG) -b v8.4 $(PB_REPO) racket/src/ChezScheme/boot/pb ; 	  else cd racket/src/ChezScheme/boot/pb && git fetch -q origin v8.4:remotes/origin/v8.4 ; fi
-	cd racket/src/ChezScheme/boot/pb && git remote set-branches origin v8.4
-	cd racket/src/ChezScheme/boot/pb && git checkout -q v8.4
+	#if [ ! -d racket/src/ChezScheme/boot/pb ] ; 	  then git clone -q $(SINGLE_BRANCH_FLAG) -b v8.4 $(PB_REPO) racket/src/ChezScheme/boot/pb ; 	  else cd racket/src/ChezScheme/boot/pb && git fetch -q origin v8.4:remotes/origin/v8.4 ; fi
+	#cd racket/src/ChezScheme/boot/pb && git remote set-branches origin v8.4
+	#cd racket/src/ChezScheme/boot/pb && git checkout -q v8.4
 pb-fetch:
-	$(MAKE) fetch-pb EXTRA_REPOS_BASE="$(EXTRA_REPOS_BASE)" PB_REPO="$(PB_REPO)" SINGLE_BRANCH_FLAG="$(SINGLE_BRANCH_FLAG)"
+	#$(MAKE) fetch-pb EXTRA_REPOS_BASE="$(EXTRA_REPOS_BASE)" PB_REPO="$(PB_REPO)" SINGLE_BRANCH_FLAG="$(SINGLE_BRANCH_FLAG)"
 pb-build:
 	cd racket/src/ChezScheme && racket rktboot/main.rkt --machine pb
 pb-stage:
diff --git a/racket/src/cs/Makefile b/racket/src/cs/Makefile
index 07599bde0d..574755d3c3 100644
--- a/racket/src/cs/Makefile
+++ b/racket/src/cs/Makefile
@@ -1,7 +1,7 @@
 # This makefile is used to a limited degree with nmake, but it mostly
 # intended for use as a GNU makefile.
 
-RACKET = ../../bin/racket
+RACKET = /workdir/tmp/usr/bin/racket
 SCHEME = scheme
 
 # Controls whether Racket layers are built as unsafe; beware that
